test/containsBy.js

```diff
@@ -0,0 +1,15 @@
+var assert = require('assert');
+
+var R = require('..');
+
+
+describe('containsBy', function() {
+
+  it('determines whether a projected list contains a projected value', function() {
```

Comment:
Is "projected" the right word? I'm trying to suggest the mapping of a domain to a codomain.